pkg/segment/tracing/utils/buildspantree.go

```diff
@@ -0,0 +1,31 @@
+package utils
+
+import (
+	"github.com/siglens/siglens/pkg/segment/tracing/structs"
+	log "github.com/sirupsen/logrus"
+)
+
+func BuildSpanTree(spanMap map[string]*structs.GanttChartSpan, idToParentId map[string]string) []*structs.GanttChartSpan {
+	results := make([]*structs.GanttChartSpan, 0)
```

Comment:
Shouldn't `results` just have one span (the root span) with all the other spans nested below that? So then the return type can be `*structs.GanttChartSpan` instead of `[]*structs.GanttChartSpan`

Reply:
Ok, I will modify that. Since the search text for gantt chart endpoint for now is fixed: `trace_id=...`